### PR TITLE
chore: add pendo analytics

### PR DESCRIPTION
**Pull request by subotic**
resolves DEV-2487
resolves DEV-2241
resolves DEV-2489

### CODE REVIEW

File: libs/vre/shared/app-analytics/src/lib/datadog-rum/datadog-rum.service.ts [hunk 1]
```diff
@@ -54,30 +59,36 @@ export class DatadogRumService {
                 });
 
                 // if session is valid: setActiveUser
+                // FIXME: so that updates to the session state are reflected in the specified user
```
Comment (mdelez) [inline libs/vre/shared/app-analytics/src/lib/datadog-rum/datadog-rum.service.ts:62]: is this a fixme for this PR?

File: libs/vre/shared/app-analytics/src/lib/pendo-analytics/pendo-analytics.service.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+@Injectable({
+    providedIn: 'root',
+})
+export class PendoAnalyticsService {
+    private config: DspInstrumentationConfig = inject(DspInstrumentationToken);
+    private session: SessionService = inject(SessionService);
+    private environment: string = this.config.environment;
+
+    constructor() {
+        // FIXME: so that updates to the session state are reflected in specified user
```
Comment (mdelez) [inline libs/vre/shared/app-analytics/src/lib/pendo-analytics/pendo-analytics.service.ts:18]: same as previous comment

**Review by mdelez — APPROVED** [review]

File: libs/vre/shared/app-analytics/src/lib/datadog-rum/datadog-rum.service.ts (hunk 1 above)
Comment (subotic) [reply libs/vre/shared/app-analytics/src/lib/datadog-rum/datadog-rum.service.ts:62]: no, for future PRs. I've created issues on Linear.

File: libs/vre/shared/app-analytics/src/lib/pendo-analytics/pendo-analytics.service.ts (hunk 2 above)
Comment (subotic) [reply libs/vre/shared/app-analytics/src/lib/pendo-analytics/pendo-analytics.service.ts:18]: no, for future PRs. I've created issues on Linear.

**Pull request merged**